### ISSUE TITLE
quality differs wide

### ISSUE THREAD

**Issue by blodone** [vuejs/awesome-vue#335]
From project to project there is no way to suggest the quality, some very good plugins and some not good ones missing lots of features u might think would work / not using best practices

**Comment by yyx990803** [comment]
We don't have enough time to evaluate every single submission on their quality, and it's kinda out of scope for this repo. I know @phanan is working on a website-version of this, maybe we should consider adding crowd-sourced rating mechanism into it.

**Comment by phanan** [comment]
Hmm. The website version, in all honesty, doesn't have a database to
"inject" the crowd-sourced rating mechanism into, and personally, I'd
prefer it to be simple that way – totally agreed with Evan that this
concern is kind of out of scope.

**Comment by miljan-aleksic** [comment]
Perhaps we could work out a guide on best practices and simply and mostly encourage reviewing those before submitting a Pull Request. The guide could have its own repo and invite everyone to participate. There is already a blog post about it from one of the 3rd party plugin developers, so I'm sure it would take no time to see collaboration on the topic.

**Comment by phanan** [comment]
To continue my comment above – since most of the resources are GitHub repositories, the stars can more or less be considered a free rating mechanism.

**Comment by Doogiemuc** [comment]
I completely agree with the OP. The "awesme lists" are great collections. But keeping them as markdown pages on github is just simply the wrong tool. We need something like stackoverflow where visitors can up/down vote entries. Stackoverflow just launched its [Stackoverlfow Documentation](http://stackoverflow.com/documentation) project. That could be a usefull replacement for these static lists.

**Comment by mgielda** [comment]
Firstly, thanks for awesome-vue, it helps a lot to get started with Vue (an awesome framework in itself).

I agree that it's better to keep things simple, since sometimes trying to fix things too hard only makes things worse!

But then I do agree it is hard to tell things apart, and wanted to suggest an improvement: couldn't there be a more unified way to show the star count, like with shields.io badges? Example below:

![badge](https://img.shields.io/github/stars/vuejs/vue.svg?label=%E2%98%85)

This would really help keeping things up to date and also this way all the repos could have a number associated with them, so that it would be a little clearer which frameworks can be considered 'widely used' and which are new kids on the block (that's not necessarily bad, of course, but definitely tells you a little about what you should expect).

I have a minor remark as well - component collections could be divided into 'collections' which are trying to be an entire 'framework' and single components (or small families of components) with no such ambition. These are two different things after all (the former are an either-or kind of choice, the latter are more like add-ins).

**Comment by mgielda** [comment]
I looked at the pull requests and I can see several people are trying to fix this already :)

#313 - just changes the star symbol (and with conflicts now anyhow, so hard to merge), I don't think this makes sense, probably ought to be closed

#429 - similar, although the link to the Github profile should be included, but changing the looks of the star for just one entry makes even less sense

#332 - I see @brookqin is doing exactly what I suggested there (using a different badge style, but the concept is the same), didn't see it at first due to my inability to read Japanese :) So perhaps it's just a matter of taking his suggestion and resolving the conflicts?

**Comment by petervmeijgaard** [comment]
@mgielda I saw the pull-request, https://github.com/vuejs/awesome-vue/pull/313, from @kazupon. He would like the have the :star: emoji instead of the standard star, ★. So I thought, I would give him a head start...

**Comment by mgielda** [comment]
OK, now I get why you did your PR like that, thanks for the explanation! Anyway I still think that it's nigh impossible to manually keep track of all the stars with such a long list so my vote goes strongly in favour of something along the lines of #332
